js/components/frame.js

```diff
@@ -521,7 +532,6 @@ class Frame extends ImmutableComponent {
     })
     this.webview.addEventListener('enter-html-full-screen', () => {
       windowActions.setFullScreen(this.props.frame, true, true)
-      setTimeout(windowActions.setFullScreen.bind(this, this.props.frame, undefined, false), 5000)
```

Comment:
didn't realize what this was for, but it looks like it turns off the fullscreen warning message so I'll put it back in